content/docs/tutorials/aws/aws-ts-stackreference.md

```diff
@@ -2,7 +2,7 @@
 title: "StackReference Example"
 ---
 
-<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/aws-ts-stackreference" target="_blank">
+<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/aws-ts-stackreference/company" target="_blank">
```

Comment:
This example shouldn't even have a button on it's README.md, because it has three separate programs in subdirs.

Reply:
I went ahead and removed the button from this example's README.md